Makefile

```diff
@@ -195,6 +197,15 @@ shell: init
 	@echo "launching a shell in the containerized build environment"
 	@$(DOCKER_RUN) /bin/sh $(CMD)
 
+.PHONY: lint-go
+lint-go:
+	@docker run												\
+		--rm 												\
+		-v "$$(pwd):/app"									\
+		-w /app												\
+		golangci/golangci-lint:$(GOLANGCI_LINT_VERSION)		\
```

Comment:
Thanks for the work! What is the difference of golangci-lint vs. the ordinary lint? I also see you have a .golangci.yml, which includes what would be checked or not checked - how is that decided?